runtime.go

```diff
@@ -219,6 +242,10 @@ func (r *runtime) InstantiateModule(
 	compiled CompiledModule,
 	mConfig ModuleConfig,
 ) (mod api.Module, err error) {
+	if e := r.failIfClosed(); e != nil {
+		return nil, e
```

Comment:
nit, but we gophers prefer `err`  over errors whenever possible.

```suggestion
	if err = r.failIfClosed(); err != nil {
		return
```

Reply:
is this fixed?

Reply:
I think I only fixed one 😬